packages/edit-site/src/components/start-template-options/style.scss

```diff
@@ -1,28 +1,27 @@
-.edit-site-start-template-options__modal {
-	.components-modal__content {
-		padding-bottom: 0;
-	}
+$actions-height: 92px;
```

Comment:
The main way the resulting layout is achieved in this PR is:

- by making the actions bar absolute positioned, so that it's always stuck at the bottom of the modal
- by using a fixed `height` (instead of padding top/bottom) to resize the actions bar
- by using that same height as extra padding bottom in the modal content, to preserve correct overflow scroll behaviour